### ISSUE TITLE
Blaze burner with straw eats buckets

### ISSUE THREAD

**Issue by restelen** [FTBTeam/FTB-Modpack-Issues#4964]
### Modpack

FTB Skies Expert

### Modpack version

1.4.1

### Describe the bug

Blaze burner with straw eats bucket or tank even when given force infused biodiesel.

### Steps to reproduce

1. Give a blaze burner a straw
2. Give the blaze burner a bucket of force infused biodisel.
3. PROFIT

### Expected behaviour

You should be getting the bucket or tank back

**Comment by TheMadTechnician** [comment]
Issue is not consistent across all fuels, but does affect different container types similarly. Biodiesel works as intended, and returns a bucket, but force infused biodiesel consumes whatever container you used to feed to blaze burner, voiding any additional liquid that container may have in it in the process. I have tried with a Mekanism basic tank in bucket mode, as well as with a Thermal reservoir with force infused biodiesel. I tested force infused diesel with the same result, but only with a bucket.
Both problematic fluids can be piped into the blaze burner without a problem, so there is a work around at least if you have the space to hook a tank up to the blaze burner.
